Makefile

```diff
@@ -3,15 +3,15 @@
 				check-testnet-data check-testnet-construction check-testnet-construction-erc20 \
 				check-mainnet-data check-mainnet-construction
 
-PROJECT             ?= avalanche-rosetta
+PROJECT             ?= camino-rosetta
 GIT_COMMIT          ?= $(shell git rev-parse HEAD)
 GO_VERSION          ?= $(shell go version | awk {'print $$3'})
 WORKDIR             ?= $(shell pwd)
-DOCKER_ORG          ?= avaplatform
+DOCKER_ORG          ?= c4tplatform
 DOCKER_IMAGE        ?= ${DOCKER_ORG}/${PROJECT}
 DOCKER_LABEL        ?= latest
 DOCKER_TAG          ?= ${DOCKER_IMAGE}:${DOCKER_LABEL}
-AVALANCHE_VERSION   ?= v1.9.3
+CAMINO_VERSION   ?= v0.2.0
```

Comment:
Can you please align formatting here so it's the same as a line above?

Reply:
fixed